appintro/build.gradle

```diff
@@ -5,17 +5,14 @@ apply plugin: 'io.gitlab.arturbosch.detekt'
 apply plugin: 'org.jlleitschuh.gradle.ktlint'
 
 group = 'com.github.AppIntro'
-
 version = "6.2.0"
 
 android {
-    compileSdkVersion 31
+    compileSdk 33
 
     defaultConfig {
-        minSdkVersion 14
-        targetSdkVersion 31
-        versionCode 20
```

Comment:
Same here

Reply:
As they're not really needed for libraries: https://developer.android.com/studio/publish/versioning